feature/profile/src/main/java/com/iteneum/profile/data/ProfileViewModel.kt

```diff
@@ -0,0 +1,99 @@
+package com.iteneum.profile.data
+
+import androidx.compose.runtime.MutableState
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.setValue
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.iteneum.Profile
+import com.iteneum.network.DataState
+import dagger.hilt.android.lifecycle.HiltViewModel
+import kotlinx.coroutines.launch
+import javax.inject.Inject
+
+/**
+ * File that contains [ProfileViewModel] that works as ViewModel for the Profile View.
+ * It will handle the data work & process when loading & be done with the view.
+ *
+ * @author Jose Miguel Garcia Reyes
+ */
+
+@HiltViewModel
+class ProfileViewModel @Inject constructor(): ViewModel() {
+    private val _profileModelMutable : MutableState<Profile> = mutableStateOf(Profile())
```

Comment:
we can change this in the same way you did with profileModel

Reply:
Ok, done!